### PR TITLE
IQX: Update contact info

### PR DESCRIPTION
**Pull request by bretg**

### CODE REVIEW

File: static/bidder-info/iqx.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: "http://rtb.iqzone.com?pid={{.SourceId}}&host={{.Host}}&pbs=1"
 maintainer:
-  email: "it@iqzone.com"
+  email: "adops@iqzone.com"
```
Comment (gargcreation1992) [inline static/bidder-info/iqx.yaml:3]: sent email for verification!

Comment (Sonali-More-Xandr) [reply static/bidder-info/iqx.yaml:3]: @bretg We have sent a verification email to `adops@iqzone.com`. Please reply back to that mail with 'Received' message to complete the verification process so we can move forward with this pull request.

Comment (bretg) [reply static/bidder-info/iqx.yaml:3]: Pinged the IQ Zone team to respond to the email

Comment (gargcreation1992) [reply static/bidder-info/iqx.yaml:3]: verified.

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**